standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/utils/MetaStoreUtils.java

```diff
@@ -105,6 +105,7 @@ protected DateFormat initialValue() {
   public static final String DB_EMPTY_MARKER = "!";
 
   public static final String EXTERNAL_TABLE_PURGE = "external.table.purge";
+  public static final String EXTERNAL_TABLE_AUTODELETE = "external.table.autodelete";
```

Comment:
After discussion with @nrg4878 and @thejasmn , we thought it's best that we don't add this new option because "external.table.purge" is widely used. Introducing the new autodelete might cause confusion.